admin-dev/themes/new-theme/js/pages/order/create/cart-provider.js

```diff
@@ -61,7 +61,7 @@ export default class CartProvider {
    */
   loadEmptyCart(customerId) {
     $.post(this.router.generate('admin_carts_create'), {
-      customer_id: customerId,
+      customerId,
```

Comment:
Intentional or mistake ?

Reply:
intentional. As we kinda on a way to use camelCase convention in api's(https://github.com/PrestaShop/PrestaShop/issues/15977#issuecomment-546274938), so I guess we must camelCase both requests and response where possible. (although it interferes bit with forms, as those are mostly named in snake_case).